vdx_vtep/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pynos>=1.3,<1.4
+netmiko
```

Comment:
We had planned to used netmiko instead of paramiko few days back but that work was pending till now, that's why netmiko is there in requirements.txt.

Now, as you also have suggested the same thing, we have updated the code and using netmiko.